test/e2e/scheduling/rescheduler.go

```diff
@@ -109,3 +113,37 @@ func podRunningOrUnschedulable(pod *v1.Pod) bool {
 	running, _ := testutils.PodRunningReady(pod)
 	return running
 }
+
+func reserveCpu(f *framework.Framework, id string, replicas, millicores int) {
+	By(fmt.Sprintf("Running RC which reserves %v millicores", millicores))
+	request := int64(millicores / replicas)
+	config := &testutils.RCConfig{
+		Client:         f.ClientSet,
+		InternalClient: f.InternalClientset,
+		Name:           id,
+		Namespace:      f.Namespace.Name,
+		Timeout:        defaultTimeout,
+		Image:          framework.GetPauseImageName(f.ClientSet),
+		Replicas:       replicas,
+		CpuRequest:     request,
+	}
+	framework.ExpectNoError(framework.RunRC(*config))
+}
+
+func CreateHostPortPods(f *framework.Framework, id string, replicas int, expectRunning bool) {
```

Comment:
This looks duplicated from `test/e2e/cluster_size_autoscaling.go`.  Move into util?